### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by GreatThingsFromHands** [LayerZero-Labs/sybil-report#143]
---
name: Sybil Report
about: Report Sybil Activity on LayerZero
title: "[Sybil Report]"
labels: under review
assignees: LayerZero-GH

---

# Reported Addresses

[Here's](https://github.com/GreatThingsFromHands/Sybil-Addresses-GreatThingsFromHands/blob/0ddcff0c1bba497d93f8d21888982f6b5afeb4ba/%23001%20Sybil%20Address%20(1769).md) is a link to all 1769 addresses (same as attached Dune query results) as submissions do not allow for more than 65536 characters.

# Description
- Cluster Size:  1769 EVM wallets, 1769 Aptos wallets
- Chain(s) of described action(s) : Polygon, Ethereum
- Methodology: Onchain sleuthing + Dune SQL query: (¹Polygon: https://dune.com/queries/3700917 / ²Ethereum: https://dune.com/queries/3701109)

Over the course of 18 days, from 2024-03-26 to 2024-04-13, 1,769¹ unique addresses executed multiple distinct actions on the Polygon network:

After churning LayerZero volume through protocols like theAptosBridge and Stargate, the attacker tried to use a privacy protocol called [Umbra](https://polygonscan.com/address/0xfb2dc580eed955b528407b4d36ffafe3da685401) to move funds (USDT) between addresses. While still considering randomizing the USDT amount (ranging from 1500 to 5650 depending on the sub-cluster that was being run), a mistake was made in the choice of which fund-mixer to use. With Umbra having near-zero concurrent users and therefore no activity, the goal of hiding one's tracks failed miserably. Not only could the money trail be easily followed on the explorer, but it also led us to unravel the whole 1,700+ strong address cluster.

![image](https://github.com/GreatThingsFromHands/L0SybilReports_GreatThingsFromHands/assets/104984062/bfd67f06-806c-49c6-8079-71960664a650)
*(The not-so-private [Umbra](https://polygonscan.com/address/0xfb2dc580eed955b528407b4d36ffafe3da685401) in- and outflows. Note the changing deposit and withdrawal addresses, while amounts remain identical (minus the ~$0.08 fee sent to the Umbra devs))*

# Detailed Methodology & Walkthrough
Over the course of 18 days, from 2024-03-26 to 2024-04-13, 1769¹ unique wallets have taken the following actions in the exact same order on the exact same day on each specific address:

1) Withdraw USDT from Umbra on Polygon
2) Bridge USDT from Polygon to Aptos via theAptosBridge
3) Bridge USDT from Aptos to Polygon via theAptosBridge
4) Deposit USDT back to Umbra on Polygon

![image](https://github.com/GreatThingsFromHands/L0SybilReports_GreatThingsFromHands/assets/104984062/ff7b991d-67a5-4567-a507-3aa51d7ba80e)
*(Reoccurring pattern moving funds in between addresses: Withdraw funds from Umbra > Bridge (slightly randomized) amount into Aptos > Bridge back out to Polygon > Deposit back into Umbra > repeat)*

Additional patterns:

Over the same 18-day period, on the Aptos network, 1444 out of the 1769¹ addresses conducted a single transaction, 266 two transactions, 44 three and the remaining 15 outliers up to six at most. Notably, the bridge back from Aptos to Polygon via TheAptosBridge constitutes **100%** of the transactions within these 1444 addresses and is present in all of the remaining 325 addresses. All activity takes place within a single day for each respective address.

![image](https://github.com/GreatThingsFromHands/L0SybilReports_GreatThingsFromHands/assets/104984062/42c490a9-51fb-4bac-8672-8aad8a8501b5)
*(A single Aptos transaction, the bridge back to Polygon, is an exact match on 1444 addresses)*

To conclude, and to further prove the affiliation of each wallet, 1741 out of the 1769 addresses have a maximum of four total Ethereum transactions, with 1246 of those addresses having precisely three. Additionally, and mainly making up these few mainnet transactions, 1547² out of the 1769 addresses bridged to zkSync Era and zkSync Lite within a three-week period, from September 10th to October 3rd, 2023. The remaining 221 addresses have either one of these bridge transactions or no ETH mainnet activity at all, while still exhibiting the detailed Polygon farming activity mentioned above.

![image](https://github.com/GreatThingsFromHands/L0SybilReports_GreatThingsFromHands/assets/104984062/3787456a-a4e4-4634-a47c-1ab4e98d1328)
*(Example ETH mainnet activity of ²)*

To summarize, the overlapping cross-chain behavior of Ethereum, Polygon, and Aptos, along with the conjunction of all these specific factors being required for any address to be considered valid in our analysis, provides statistically sound evidence that all of these addresses are controlled by a single entity. An entity which is not only farming LayerZero, but zkSync as well. Any statements and claims made in this report are welcomed to be verified by checking the SQL queries used to extract the cluster. 

- ¹ Polygon: https://dune.com/queries/3700917
- ² Ethereum: https://dune.com/queries/3701109

# Reward Address (If Eligible)
0x95acE0e9a722644a020210a7429293Aa629E605D
